scripts/swolisten.c

```diff
@@ -435,10 +435,36 @@ int usbFeeder(void)
       if (!(dev = libusb_get_device(handle)))
 	continue;
 
-      if (libusb_claim_interface (handle, INTERFACE)<0)
+      uint8_t bmp_interface = 0;
+      unsigned char bmp_endpoint = 0;
```

Comment:
We would suggest using `uint8_t` for this variable too as that then matches to the type for bEndpointAddress and removes an item needing modernisation.

Reply:
Accepted. I was first looking at signature of `libusb_bulk_transfer()`, and it's `unsigned char endpoint` there.
```c
int LIBUSB_CALL libusb_bulk_transfer(libusb_device_handle *dev_handle,
	unsigned char endpoint, unsigned char *data, int length,
	int *actual_length, unsigned int timeout);
```